packages/langium/test/lsp/go-to-type.test.ts

```diff
@@ -0,0 +1,87 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
```

Comment:
This file should be `signatureHelpProvider.test.ts`. Also:
```suggestion
 * Copyright 2022 TypeFox GmbH
```